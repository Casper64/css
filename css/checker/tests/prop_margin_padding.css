
.margin {
    margin: 10px 20px 30px 40px;
    margin: 10px 20px;
    margin: 100px;
    margin: 10px 20px 30px;
}

.padding {
    padding: 10px 20px 30px 40px;
    padding: 10px 20px;
    padding: 100px;
    padding: 10px 20px 30px;
}

.unsupported {
    padding-inline-start: 10px;
    margin-block-end: 10%;
}

.too_many {
    padding: 10px 20px 30px 40px 50px;
    margin: 10px 20px 30px 40px 50px;
}