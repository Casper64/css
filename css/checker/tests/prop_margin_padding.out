css/checker/tests/prop_margin_padding.css:17:5: error: unsupported property "padding-inline-start"! Check the "CAN_I_USE.md" for a list of supported properties
   15 | 
   16 | .unsupported {
   17 |     padding-inline-start: 10px;
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~
   18 |     margin-block-end: 10%;
   19 | }
css/checker/tests/prop_margin_padding.css:18:5: error: unsupported property "margin-block-end"! Check the "CAN_I_USE.md" for a list of supported properties
   16 | .unsupported {
   17 |     padding-inline-start: 10px;
   18 |     margin-block-end: 10%;
      |     ~~~~~~~~~~~~~~~~~~~~~~
   19 | }
   20 |
css/checker/tests/prop_margin_padding.css:22:34: error: property "padding" can have a maximum of 4 values
   20 | 
   21 | .too_many {
   22 |     padding: 10px 20px 30px 40px 50px;
      |                                  ~~~~
   23 |     margin: 10px 20px 30px 40px 50px;
   24 | }
css/checker/tests/prop_margin_padding.css:23:33: error: property "margin" can have a maximum of 4 values
   21 | .too_many {
   22 |     padding: 10px 20px 30px 40px 50px;
   23 |     margin: 10px 20px 30px 40px 50px;
      |                                 ~~~~
   24 | }